README.md

```diff
@@ -1,68 +1,72 @@
 # Google Ads Doctor - oauthdoctor
-This program will verify your Google Ads client oauth environment and report
+This program will verify your Google Ads client OAuth2 environment and report
 anomalies. Where possible, it will guide you through correcting the problems.
 This assumes that you have already completed the steps in your client library
 [README.md](https://developers.google.com/google-ads/api/docs/first-call/get-client-lib)
 file.
 
 The program vets your client library configuration file, verifies connectivity
-to the Google Ads host, and simulates either the installed application or web
+to the Google Ads host, and executes either the installed application or web
 flow OAuth2 process.
 
 If it discovers errors it will attempt to guide you to correct them. For example,
 if your refresh token is invalid, it will walk you through the process of
 creating a valid token which is then written to your client library configuration
 file.
 
-# Requirements
-Go minimum version 1.11. We require 1.11 or greater because we are using
-[Go modules](https://github.com/golang/go/wiki/Modules) for dependency management.
-
-# Setup
-To run the program, you first need to install the Go programming language
-runtime. Download the [latest version](https://golang.org/dl/) and follow
-the installation instructions.
+# Downloads
+If you are building from source, follow the
+[source installation instructions](#source).
 
-Clone the repository outside of your GOPATH. If you must clone within your GOPATH
-then set the environment variable `GO111MODULE=off`
+If you want to use a binary, fetch the repository and
 
-# Running the program
-Once you have verified your Go installation, in a terminal, change to the
-directory where you cloned the repository. Change to the directory
-oauthdoctor.
+```
+cd <path>/google-ads-doctor/oauthdoctor/bin
+```
 
-At the command line, type:
+There are three directories, one each for Darwin (OSX), Linux, and Windows.
+You can run the binary in place or copy the file to any location on your system.
```

Comment:
Suggest rewriting the sentence to:

"You can run the executable file in place or..."

Reply:
I searched for other project and they refer to these files as binaries and not executables.

https://golang.org/dl/ refers to the pre-build releases as binaries.